src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -100,7 +99,6 @@ interface WorkspaceContainerProps extends PropsWithChildren {
   refresh: () => Promise<void>;
   workspace: Workspace;
   refreshWorkspace: () => void;
-  silentlyRefreshWorkspace: (errorHandling?: ErrorCallback) => Promise<void>;
```

Comment:
This is no longer a necessary parameter now that we're doing the deletion polling with a separate hook.

I didn't remove it entirely in this PR, since removing it from the useWorkspace hook would effect a sizable number of test files across the project, and unnecessarily increase the size of the diff for this PR.

Reply:
Do you plan to completely remove it in a follow-up PR?

Reply:
I'm thinking not for this ticket, but for [WOR-1338](https://broadworkbench.atlassian.net/browse/WOR-1338) instead, since we'll already likely have other teams involved (IIRC, AJ is also the owner of the other tests that are mocking this hook).